packages/document/module-doc/docs/en/guide/faq/basic.mdx

```diff
@@ -0,0 +1,11 @@
+# General Questions
+
+## What is the relationship between Modern.js Module and Modern.js Builder?
+
+Modern.js Module uses esbuild to build toolkits and component libraries, and Modern.js Builder focuses on solving web application building scenarios.
+
+## Can Modern.js Module use webpack plugins or loaders?
+
+Modern.js Module is based on esbuild for building and cannot use tools from the webpack-related ecosystem.
```

Comment:
add some info for esbuild plugins? Users may find some esbuild plugins as alternative.